index.js

```diff
@@ -1,19 +1,41 @@
 // simple node web server that displays hello world
 // optimized for Docker image
 
-var express = require('express');
+const express = require('express');
 // this example uses express web framework so we know what longer build times
 // do and how Dockerfile layer ordering matters. If you mess up Dockerfile ordering
 // you'll see long build times on every code change + build. If done correctly,
 // code changes should be only a few seconds to build locally due to build cache.
 
-var morgan = require('morgan');
+const morgan = require('morgan');
 // morgan provides easy logging for express, and by default it logs to stdout
 // which is a best practice in Docker. Friends don't let friends code their apps to
 // do app logging to files in containers.
 
-// Appi
-var app = express();
+const MongoClient = require('mongodb').MongoClient;
+// this example includes a connection to MongoDB
+
+const {
```

Comment:
@BretFisher docker-compose supports reading [env variables from a file](https://docs.docker.com/compose/environment-variables/#the-env_file-configuration-option). Would that be sufficient instead of adding a separate file for every env variable? I my self dont have experience with docker secrets.

Reply:
The point of secrets is to they *don't* get stored as environment variables if they don't have to... so no, you can't use a env file to store secrets. Let me take another pass as this as-is and we can make secrets support a follow-up PR.